mirror_maker.go

```diff
@@ -265,6 +273,19 @@ func (this *MirrorMaker) startProducers() {
 	}
 }
 
+func (this *MirrorMaker) startMetricsProducer() {
```

Comment:
what if this dies, maybe we need some go routines and channel state moving around. we can have an http endpoint that pings into that go routine that is watching/monitoring everything to make sure "a ok" else the process gets killed.